numpy/core/tests/test_matrix_transpose.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from hypothesis import given
+from hypothesis.extra.numpy import array_shapes
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+
+def test_matrix_transpose_equals_swapaxes_1d():
```

Comment:
Here and below the test name needs to be renamed to use transpose, not swapaxes.